src/Gui/Application.cpp

```diff
@@ -1914,8 +1914,8 @@ void Application::runApplication()
         // opens them
         QDir cwd = QDir::current();
         std::list<std::string> files = App::Application::getCmdLineFiles();
-        for (std::list<std::string>::iterator jt = files.begin(); jt != files.end(); ++jt) {
-            QString fn = QString::fromUtf8(jt->c_str(), static_cast<int>(jt->size()));
+        for (const auto &file : files) {
+            QString fn = QString::fromUtf8(file.c_str(), static_cast<int>(file.size()));
```

Comment:
Sort of an aside, but why in the world are we providing a size here? Feels more like an opportunity to introduce an error than to prevent one.